test/test.ts

```diff
@@ -427,3 +427,70 @@ test('can open a specific channel twice', async t => {
   const ch2 = await conn.channel(2)
   t.is(ch2, ch)
 })
+
+
+test('set basic flow on channel', async t => {
+  const amqp = new AMQPClient("amqp://127.0.0.1")
+  const conn = await amqp.connect()
+  const ch = await conn.channel()
+  await t.notThrowsAsync(async () =>  await ch.basicFlow(true))
```

Comment:
doesn't this throw with rabbitmq? or is it just `basicFlow(false)` that does?

Reply:
`basicFlow(false)` throws.